[android] Set Badge Count Number Based on Notification Payload (#17171)
expo/expo: CHANGELOG.md

- Updated `@stripe/stripe-react-native` from `0.2.3` to `0.6.0`. ([#16938](https://github.com/expo/expo/pull/16938) by [@kudo](https://github.com/kudo))
- Updated `react-native-reanimated` from `2.4.1` to `2.8.0`. ([#16956](https://github.com/expo/expo/pull/16956), [#17159](https://github.com/expo/expo/pull/17159) by [@tsapeta](https://github.com/tsapeta))
- Updated `react-native-maps` from `0.29.4` to `0.30.1`. ([#16944](https://github.com/expo/expo/pull/16944) by [@bycedric](https://github.com/bycedric))

> - Set badge count number on android by fetching the value from the notification payload content. ([#17171](https://github.com/expo/expo/pull/17171) by [@danstepanov](https://github.com/danstepanov))

Reviewer: Move this entry to the `packages/expo-notifications/CHANGELOG` and place it under [`Unpublished -> Bug fixes` section](https://github.com/expo/expo/blob/main/packages/expo-notifications/CHANGELOG.md#-bug-fixes)
```suggestion
- On Android fix the `badge count number` not being read from the notification payload content. ([#17171](https://github.com/expo/expo/pull/17171) by [@danstepanov](https://github.com/danstepanov))
```

Reviewer 2: I also recommend reading this guide [Updating Changelogs](https://github.com/expo/expo/blob/main/guides/contributing/Updating%20Changelogs.md) for more details on how we use changelogs 😉

Author: yea was late, forgot to move the changelog, thanks for flagging this
